src/news/controller/dtos/news.update.request.output.ts

```diff
@@ -0,0 +1,18 @@
+import { IsString } from "class-validator";
+import PartialClass from "../../../domain/base/partial.class";
+import { ApiProperty } from "@nestjs/swagger";
+import UpdateNewsUseCaseOutuput from "../../usecases/dtos/update.news.usecase.output";
+
+export default class NewsUpdateRequestOutPut extends PartialClass {
```

Comment:
Essa classe deve se chamar `UpdateNewsControllerOutput` e deve estender a classe `CreateNewsControllerOutput` assim como o exemplo de voluntário 

https://github.com/the-web-forest/webforest-api/blob/develop/src/volunteer/controller/dtos/update.volunteer.controller.output.ts